CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 - Fixed Youtube loading screen stuck issue (#2031)
+- Fixed GreyScreen issue (#1940)
```

Comment:
I don't know if users will understand what that means.  What about `Fixed bug that could cause websites to show up as blank, grey screens when resuming Firefox`